packages/dbus-broker/dbus-broker.service

```diff
@@ -2,7 +2,9 @@
 Description=D-Bus System Message Bus
 Documentation=https://github.com/bus1/dbus-broker
 DefaultDependencies=false
-After=dbus.socket
+# Ensure the dbus user is created before starting dbus-broker
+After=dbus.socket systemd-sysusers.service
```

Comment:
You will have to make the same change to `whippet.service` now that I merged #661 :sweat_smile:

Reply:
Applied same changes to whippet.service as it is also aliased as dbus.service.